test/tests.zig

```diff
@@ -1274,9 +1275,16 @@ pub fn addCAbiTests(b: *std.Build, skip_non_native: bool, skip_release: bool) *S
                 .pic = c_abi_target.pic,
                 .strip = c_abi_target.strip,
             });
+
+            const c_flags: []const []const u8 = if (target.cpu.arch == .wasm32 and optimize_mode == .Debug)
```

Comment:
Rather than setting the optimization mode differently for Wasm, it'd be better to have a smaller test case for boolean vectors just for the Wasm target (preferably with a comment as to why in the code) until we run the c-abi tests for the self-hosted Wasm backend. We still want to ensure the generated code is correct for debug mode, also.